intel_extension_for_transformers/llm/runtime/graph/application/main_pybind.cpp

```diff
@@ -492,6 +492,10 @@ PYBIND11_MODULE(polyglot_cpp, m)
 
 PYBIND11_MODULE(mistral_cpp, m)
 
+#elif MODEL_NAME_ID == 15
```

Comment:
https://github.com/intel/intel-extension-for-transformers/blob/94697ef03c3ba030353a25061fcb2582123fef95/intel_extension_for_transformers/llm/runtime/graph/application/main_run.cpp#L234-L238

Since this qwen also use a specialized tokenizer, do you need to modify `main_run.cpp` just like chatglm?